### DISCUSSION TITLE
Support interface fan speed

### DISCUSSION THREAD

**Discussion by jmiller1111** [OrcaSlicer/OrcaSlicer#5364] [Ideas]
Hello,

I've been having serious problems with supports -- difficult to remove, fuzed to the print, etc..  Stumbled across a cooling setting "Support interface fan speed" in the filament config menu.  The dropdown says that "can only be overridden by the "disable fan first layers".

When using filaments like PET and Nylons, the manufacturers say to turn the fan off.  Basically, no fan for first 1000 layers.  So, this means cooling the support interface is not available for filaments that print with fan off?  On a hunch, I ignored the manufacturers instructions for "turn fan off" and set the support interface fan speed" to 75%.  The support was much easier to remove but I'm sure the part is significantly weaker because of using the fan for the whole print.

How about letting the "Support interface fan speed" override the "disable fan first layers" for these filaments??

**Comment by coryrc** [comment]
Can you set min and max fan to 0% instead? or set "Full Speed At Layer" to something very high like 2000 or 4000, while setting "disable fan first layers" to just below your lowest interface?

**Comment by knottywood440** [comment]
If we have "no cooling for the first" 3 layers and  "Full Speed At Layer" 0 as our settings, will "full fan speed at layer" defer to our min and max fan speed threshold at layer 4?
